src/elm/Claim.elm

```diff
@@ -291,19 +359,66 @@ viewClaimCard { selectedCommunity, shared, accountName } openConfirmationModalMs
                 div [ class "flex justify-between space-x-4" ]
                     [ button
                         [ class "button button-danger"
-                        , onClick (openConfirmationModalMsg claim.id False)
+                        , Utils.onClickNoBubble (OpenVoteModal claim.id False)
                         ]
                         [ text (t "dashboard.reject") ]
                     , button
                         [ class "button button-primary"
-                        , onClick (openConfirmationModalMsg claim.id True)
+                        , Utils.onClickNoBubble (OpenVoteModal claim.id True)
                         ]
                         [ text (t "dashboard.verify") ]
                     ]
             ]
         ]
 
 
+viewPhotoModal : LoggedIn.Model -> Model -> Html Msg
+viewPhotoModal loggedIn claim =
+    let
+        { t } =
+            loggedIn.shared.translators
+
+        body =
+            [ div [ class "md:flex md:justify-start md:space-x-4" ]
+                [ img [ style "max-height" "42vh", src "http://cambiatus.miskov.ru/trash.png" ] []
+                , div []
+                    [ label [ class "mt-6 md:mt-0 input-label md:text-xl block" ]
+                        [ text "verification number"
+                        ]
+                    , strong [ class "text-xl md:text-3xl" ] [ text "82378463" ]
+                    ]
+                ]
+            ]
+
+        withPhotoModalFooter =
+            if isValidated claim loggedIn.accountName then
```

Comment:
we shouldn't show the buttons if the user is not a validator as well, this way anyone can check the status of the claim, even if its not a validator

Reply:
Yes, indeed! I've done it on the View Claim but forgot to add here... Should be fixed now!